packages/ui/src/components/Menu/index.tsx

```diff
@@ -173,9 +163,8 @@ const MenuList = styled.div<MenuListProps>`
 const FwdMenu = forwardRef(
   (
     {
-      align = { left: '50%', right: 'inherit' },
       ariaLabel = 'Menu',
-      baseId = 'menu',
+      id = 'menu',
```

Comment:
Default value should be removed as it's an idea we can have side effects that many modals has the same id

```suggestion
      id,
```